openhtf/plugs/usb/__init__.py

```diff
@@ -52,10 +58,18 @@ def _open_usb_handle(**kwargs):
   Returns:
     Instance of UsbHandle.
   """
-  if conf.Config().usb_server:
-    return None
-  else:
-    return local_usb.LibUsbHandle.Open(**kwargs)
+
+  serial = None
+  remote_usb = conf.Config().remote_usb.strip()
+  if (remote_usb == 'EtherSync'):
+    device = conf.Config()[remote_usb]
+    if device and isinstance(device, dict) and device.has_key('mac_addr') and device.has_key('plug_port'):
```

Comment:
Instead of the 3-conjunct test here, you might want to just put this stuff into a 'try' block and raise the appropriate exception when you catch a KeyError.